### PR TITLE
Add role & description

### PR DESCRIPTION
**Pull request by Coraellation**

### CODE REVIEW

**Comment by Coraellation** [comment]
@davepagurek PTAL :)

File: 2016/data/team.json [hunk 1]
```diff
@@ -62,7 +62,9 @@
     },
     {
       "name": "Cora Lam",
-      "image": "img/people/cora.jpg"
+      "image": "img/people/cora.jpg",
+      "roles": ["Logistics", "Community", "Volunteers"],
+      "description": "A human. Being."
```
Comment (yucombinator) [inline 2016/data/team.json:67]: So deep

**Comment by yucombinator** [comment]
lgtm!

File: 2016/data/team.json (hunk 1 above)
Comment (Coraellation) [inline 2016/data/team.json:67]: What is life. What is love.

**Pull request merged**